lib/assignDisabledRanges.js

```diff
@@ -253,8 +253,8 @@ module.exports = function (root, result) {
 					});
 				}
 
-				Object.keys(disabledRanges).forEach((ruleName) => {
-					if (!_.get(_.last(disabledRanges[ruleName]), 'end')) {
+				Object.entries(disabledRanges).forEach(([ruleName, ranges]) => {
+					if (!_.get(ranges[ranges.length - 1], 'end')) {
```

Comment:
[question] What do you think about using the `slice(-1)` idiom? Is it a bit tricky?

```suggestion
					if (!_.get(ranges.slice(-1), 'end')) {
```

See also <https://learnersbucket.com/examples/array/how-to-get-the-last-element-of-the-array-in-javascript/>

Reply:
`array.slice(-1)` returns an array with one element. We'd have to use `array.slice(-1)[0]`, which I don't think is as readable as `array[array.length - 1]`.

Reply:
Thanks. It makes sense. 👍🏼